_targets.R

```diff
@@ -1,17 +1,16 @@
 library(targets)
 library(tarchetypes)
-library(dataRetrieval)
-library(EflowStats)
 library(tidyverse)
```

Comment:
`library(tidyverse)` can be deleted because you're loading it below with `suppressPackageStartupMessages(library(tidyverse))`